client/lib/perfmon/index.js

```diff
@@ -0,0 +1,178 @@
+/** 
+ * Internal dependencies
+ */
+import analytics from 'analytics';
+
+/**
+ * External dependencies
+ */
+import each from 'lodash/collection/each';
+import remove from 'lodash/array/remove';
+import debounce from 'lodash/function/debounce';
+import page from 'page';
+
+var debug = require( 'debug' )( 'calypso:perfmon' );
+
+const PLACEHOLDER_CLASSES = [
+	'placeholder',
+	'pulsing-dot is-active',
+	'is-loading'
+];
+
+const PLACEHOLDER_MATCHER = PLACEHOLDER_CLASSES.map(function(clazz) { return `[class*='${clazz}']`; }).join(', ');
+const OBSERVE_ROOT = document.getElementById('wpcom');
+
+let activePlaceholders = [];
+let placeholdersVisibleStart = null;
+
+var perfmon = {
+
+	init: function() {
+		var MutationObserver = window.MutationObserver || window.WebKitMutationObserver;
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/commit/72b498ca9b1cf8ab5466617c71de7ee5e50f0646